src/js/edu-benefits/components/benefits-eligibility/BenefitsSelectionReview.jsx

```diff
@@ -9,8 +9,8 @@ export default class BenefitsSelectionReview extends React.Component {
       <table className="review usa-table-borderless">
         <tbody>
           <tr>
-            <td>Post-9/11 GI Bill (Chapter 33)</td>
-            <td>{this.props.data.chapter33 ? 'Yes' : 'No'}</td>
+            <td>Benefits selection:</td>
```

Comment:
When testing this locally, the relinquishment date isn't appearing in the Review section (I entered a future date).